optuna/samplers/_tpe/sampler.py

```diff
@@ -76,7 +76,7 @@ class TPESampler(BaseSampler):
 
     For further information about TPE algorithm, please refer to the following papers:
 
-    - `Algorithms for Hyper-Parameter Optimization
+    - `Algorithms for Hyperparameter Optimization
```

Comment:
This `Hyper-Parameter` is in the title of the [paper](https://papers.nips.cc/paper/2011/file/86e8f7ab32cfd12577bc2619bc635690-Paper.pdf), and we cannot change it. Could you revert this change?
```suggestion
    - `Algorithms for Hyper-Parameter Optimization
```